beginner_source/blitz/neural_networks_tutorial.py

```diff
@@ -55,16 +55,30 @@ def __init__(self):
         self.fc2 = nn.Linear(120, 84)
         self.fc3 = nn.Linear(84, 10)
 
-    def forward(self, x):
-        # Max pooling over a (2, 2) window
-        x = F.max_pool2d(F.relu(self.conv1(x)), (2, 2))
-        # If the size is a square, you can specify with a single number
-        x = F.max_pool2d(F.relu(self.conv2(x)), 2)
-        x = torch.flatten(x, 1) # flatten all dimensions except the batch dimension
-        x = F.relu(self.fc1(x))
-        x = F.relu(self.fc2(x))
-        x = self.fc3(x)
-        return x
+    def forward(self, input):
+        # Convolution layer C1: 1 input image channel, 6 output channels,
+        # 5x5 square convolution, it uses RELU activation function => 6x28x28 tensor
```

Comment:
```suggestion
        # 5x5 square convolution, it uses RELU activation function and outputs a Tensor with size (6, 28, 28)
```

This feels clearer to me? If you agree we could update this and the ones below.

Reply:
Maybe even `(B, 6, 28, 28)` if you want to show how the batch appears and how it is impacted by the flatten op below.